docs/spec.md

```diff
@@ -435,6 +435,8 @@ tensor. Depending on the element type, does the following:
 // %result: [2, 0, 2]
 ```
 
+&nbsp;[More Examples](../stablehlo/tests/interpret_abs.mlir)
+
```

Comment:
https://github.com/openxla/stablehlo/issues/1083 looks very relevant to this PR. Let's update the spec to fix the output type and change StablehloOps.td to follow that.